src/python_testing/TC_SC_3_5.py

```
@@ -165,8 +183,13 @@ async def open_commissioning_window(self):
     async def reopen_commissioning_window(self):
         ''' Before reopening Commissioning Window, we need to instruct TH_Server to revoke any active OpenCommissioningWindows '''
 
-        revokeCmd = Clusters.AdministratorCommissioning.Commands.RevokeCommissioning()
-        await self.th_client.SendCommand(nodeId=self.th_server_local_nodeid, endpoint=0, payload=revokeCmd, timedRequestTimeoutMs=9000)
+        try:
+            revokeCmd = Clusters.AdministratorCommissioning.Commands.RevokeCommissioning()
+            await self.th_client.SendCommand(nodeId=self.th_server_local_nodeid, endpoint=0, payload=revokeCmd, timedRequestTimeoutMs=9000)
+        except InteractionModelError as e:
+            asserts.assert_not_equal(e.clusterStatus, Clusters.AdministratorCommissioning.Enums.StatusCode.kWindowNotOpen,
```

should we re-throw the error?

Old code would "throw exception on error".
New code throws exception and "ignore all errors execept kWindowNotOpen". Why this change? Since we are ignoring errors without logging, there should be a comment here explaining why such errors are ignored.

Another part: what is the code path that is expected? if we expect an error, we should have an assert failue after the await above. Right now both paths are acceptable in the code:
   - SendCommand without an error is fine
   - SendCommand with any error except kWindowNotOpen is fine
   
This feels off.

---

- ok good catch, i need to handle all other errors.
- so you are right, this is very confusing, I meant for that to be a 3-in-1 method....the assertion was to ENSURE that the commissioning did not complete in previous step. 

- So I fixed the method and added a `def assert_dut_commissioner_failed_to_complete_commissioning` to do that job of ensuring commissioning failed.

---

i will also refactor the method, completely (I will make it independent from previous test steps)